common/containers/Tabs/ENS/components/NameResolve/components/BidModal.scss

```diff
@@ -0,0 +1,75 @@
+@import 'common/sass/variables';
+
+$summary-height: 54px;
+$button-break: 'max-width: 620px';
+
+.BidModal {
+  min-width: 580px;
+
+  @media (#{$button-break}) {
+    min-width: 0;
+  }
+
+  &-summary {
+    display: flex;
+    margin-bottom: 30px;
+
+    &-icon {
+      height: $summary-height;
+      width: $summary-height;
+      margin: 0 30px;
+    }
+
+    &-amount {
+      flex-grow: 1;
+      text-align: center;
+      line-height: $summary-height / 2;
+
+      &-arrow {
+        display: inline-block;
+        height: $summary-height / 3;
+        width: 100%;
+        background-image: url('~assets/images/icon-dot-arrow.svg');
+        background-size: auto 100%;
+        background-repeat: no-repeat;
+        background-position: center;
+      }
+
+      &-currency {
+        color: $brand-danger;
+      }
+    }
+  }
+
+  &-details {
+    padding-left: 30px;
+    margin-bottom: 20px;
+  }
+
+  &-confirm {
+    text-align: center;
+    font-weight: bold;
+    font-size: $font-size-medium-bump;
+  }
+
+  &-loading {
+    text-align: center;
+    font-size: $font-size-medium-bump;
+  }
+}
+
+// Modal overrides for extra long buttons
+@media (#{$button-break}) {
+  .BidModalWrap {
+    .Modal-footer-btn {
+      display: block;
+      float: none;
+      width: 100%;
+      margin: 0 0 5px;
+
+      &:last-child {
+        margin: 0;
+      }
+    }
+  }
+}
```

Comment:
Given that this is the second time we've needed this, I think we need a better solution. Don't worry about it here, just a note for the future.